.github/workflows/ci.yml

```diff
@@ -335,6 +336,7 @@ jobs:
       - name: Integration 1
         env:
           CGO_ENABLED: 1
+          ENABLE_CRI_SANDBOXES: ${{ matrix.enable_cri_sandboxes }}
```

Comment:
```suggestion
          if( '${{ matrix.enable_cri_sandboxes }}' -eq 'sbserver' {
                    CONTAINERD_ENABLE_CRI_SANDBOXES: ${{ matrix.enable_cri_sandboxes }}
          }
```

Reply:
unset when not sandboxed?

Reply:
We can do that.
I prefer to keep CI yaml files minimal (as it's already blown).
We'll have to add this check in like 7 steps.
I think of this as a temporary switch until we figure out how to switch between CRI servers.